src/renderers/WebGLRenderer.js

```diff
@@ -653,24 +658,24 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	var deallocateTexture = function ( texture ) {
 
-		if ( texture.image && texture.image.__webglTextureCube ) {
+		if ( texture.image && objectRendererWebGLProps[texture.uuid].__image__webglTextureCube ) {
```

Comment:
There are a lot of unconditional access to `objectRendererWebGLProps[....uuid].prop`.

Could `objectRendererWebGLProps[....uuid]` ever return undefined?